WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -937,7 +937,7 @@ local methods = {
                     self.frame:SetScript("OnClick", nil)
                 end
             end
-        else
+        elseif (self.dragging) then
```

Comment:
Right, that's already enough. The problem is that this was previously calling UpdateButtonsScroll() for each aura.